src/pybind/mgr/localpool/module.py

```diff
@@ -0,0 +1,80 @@
+from mgr_module import MgrModule, CommandResult
+import json
+import threading
+
+class Module(MgrModule):
+    def __init__(self, *args, **kwargs):
+        super(Module, self).__init__(*args, **kwargs)
+        self.stop = False
+        self.serve_event = threading.Event()
+
+    def notify(self, notify_type, notify_id):
+        self.log.info('got notify type %s id %s' % (notify_type, notify_id))
```

Comment:
I think it was probably me that started leaving these log calls on notify in the python code, but let's degrade this to a .debug() or remove it to avoid being too spammy